collections/_posts/2023-11-09-Elongated-structures.md

```diff
@@ -0,0 +1,355 @@
+ 
+**Representation of a curve as the image of a parameter function** :  
+
+The position of a curve $$\tilde{c} = c(t)+ z(t)$$
+with the parameter variable $$t ∈[\underset{0}{t}, \underset{n}{t}]$$ is the sum of the position of the curve $$c(t)$$ superposed by a noise term $$z(t)$$ with probability density function (pdf)
+
+- traces a noisy image of the curve
+- can be used as an optimization target for CNN training 
+
+**Description of a curve as a level set**
+
+$$f(x)$$ is a measure for the distance of a point x to the curve : \\
+$$(1) \quad C = \{x \in \mathbb{R}^n : f(x) = 0\}$$ 
+
+$$D$$ is the distribtion of a curve : \\
+$$(2) \quad D(x) = \delta(f(x))$$ 
+- with $$f(x)$$ the distance function, in the space $$\mathbb{R}^n$$
+- with $$\delta(s)$$, $$s \in \mathbb{R}$$ the Dirac function. 
+$$\delta(t) = \begin{cases}
+    0 & \text{pour } t \neq 0 \\
+    \infty & \text{pour } t = 0
+\end{cases}$$
+
+Now it is possible to convolve this distribution with a filter function $$w(x)$$ and obtain the heatmap :
+$$(3) \quad \begin{align*}
+H(x) &= D(x) * w(x) \\
+     &= \int_{\mathbb{R}^2} \delta(f(y)) w(x - y) dy \\
+     &= \int_C w(x - y) dy
+\end{align*}$$
+
+To be faster than while using a numerical integration and to generalize the description, $$H(x)$$ is simlified. They make the assumption that $$w(x)$$ is symetric $$w(x) = w(y)$$ for all $$\vert \vert x \vert \vert \underset{2}{} = \vert \vert y \vert \vert \underset{2}{}$$.
+
+If $$C$$ is an infinite straight line, the evaluation of the integral simplifies to $$h : \mathbb{R} \rightarrow \mathbb{R}$$ which only depends on the distance to the line C : 
+
+$$(4) \quad H(x)= h(f(x))$$
+
+(4) will be used in the general case as an approximation for the integral in (3).
+
+## Numerical approximation and implementation
+
+![](/collections/images/elongated_structures/fig_2.jpg)
+
+Here are the different steps of this approximation of a convolution by a distance dependent function : 
+
+- **data point :** description of the original structure with a finite set of data points
+- **curve approximation** : the data points are used to derive a functional curve approximation by performing a smooth parametric B-spline interpolation
+- **restricting calculation** to a “compact hull” or close region to the curve because only spatial positions within a limited distance to the curve are relevant
+- **sampling the interpolating function** : selecting a set of points from the B-spline curve created
+- **form a 2D Polyline** : sequence of connected line segments, approximation of the original curve.
+- **bounding box around each segment** (minimum enclosing box)
+- **calculate distance value for each spatial position inside** : distance from the spatial position to the original curve
+- **heatmap calculation** : resolve multiple distance estimates from overlapping bounding boxes then evaluate the distance values with the distance dependent function.
+
+**Curve parameterization**
```

Comment:
Is this the following of the previous list ? If not maybe it would be clearer to add a small sentence to indicate what is going on